pkg/dockerregistry/server/simplelookupicsp.go

```diff
@@ -17,15 +20,18 @@ import (
 // *Context AlternativeBlobSourceStrategy interface.
 type simpleLookupICSP struct {
 	icspClient operatorv1alpha1client.ImageContentSourcePolicyInterface
+	idmsClient cfgv1client.ImageDigestMirrorSetInterface
 }
 
 // NewSimpleLookupICSPStrategy returns a new entity of simpleLookupICSP using provided client
 // to obtain cluster wide ICSP configuration.
```

Comment:
Can you update this function's comment so that it reflects the fact that the configuration may also come from the IDSM?